llama-index-integrations/vector_stores/llama-index-vector-stores-azurecosmosnosql/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.vector_stores.azurecosmosnosql"
+
+[tool.llamahub.class_authors]
+AzureCosmosDBMongoDBVectorSearch = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index vector_stores azurecosmosnosql integration"
+exclude = ["**/BUILD"]
+license = "MIT"
+name = "llama-index-vector-stores-azurecosmosnosql"
+readme = "README.md"
+version = "0.1.3"
```

Comment:
nit: can we use something like `0.1.0` or even `1.0.0` since this is a brand new integration?

Reply:
Updated